internal/venice-common/src/test/java/com/linkedin/venice/helix/HelixReadOnlySchemaRepositoryTest.java

```diff
@@ -185,7 +185,7 @@ public void getSupersetOrLatestValueSchemaTest(boolean wcEnabled, boolean aaEnab
     ZkClient zkClient = mock(ZkClient.class);
     HelixSchemaAccessor accessor = mock(HelixSchemaAccessor.class);
     HelixReadOnlySchemaRepository schemaRepository =
-        new HelixReadOnlySchemaRepository(storeRepository, zkClient, accessor, 10, 100);
+        new HelixReadOnlySchemaRepository(storeRepository, zkClient, accessor, 10, 10000);
```

Comment:
Can you confirm that this change is necessary? Otherwise, I'm good with the PR. Please add to PR's description about the rationale in bumping retry count. Thank you!